src/Stratis.Bitcoin/Consensus/ConsensusManager.cs

```diff
@@ -31,7 +32,7 @@ public class ConsensusManager : IConsensusManager
         private const long ConsumptionThresholdBytes = MaxUnconsumedBlocksDataBytes / 10;
 
         /// <summary>The maximum amount of blocks that can be assigned to <see cref="IBlockPuller"/> at the same time.</summary>
-        private const int MaxBlocksToAskFromPuller = 5000;
+        private const int MaxBlocksToAskFromPuller = 30000;
```

Comment:
Are we sure this is needed? memory seems to be very big after downloading for a while and blocks are bottle queued in validation.
I guess we will know more when we have better stats.

Reply:
There is a limit of 200mb of block data that can be cached and waiting for being validated. We will never download and keep unvalidated more than that. 

I increased BP max blocks because apparently it can download much more per seconds which we didn't know before.